nav2_behavior_tree/plugins/decorator/distance_controller.cpp

```diff
@@ -35,17 +36,16 @@ DistanceController::DistanceController(
   const BT::NodeConfiguration & conf)
 : BT::DecoratorNode(name, conf),
   distance_(1.0),
-  global_frame_("map"),
-  robot_base_frame_("base_link"),
   first_time_(false)
 {
-  getInput("distance", distance_);
```

Comment:
Why remove distance?

Reply:
Mistake on my side, will revert it back.